ux.symfony.com/templates/cookbook.html.twig

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html.twig' %}
+
+{% set meta = {
+    title: 'Cookbook',
+    title_suffix: ' - Symfony UX',
+    description: 'Symfony UX cookbook - Concrete exemple to understant all the concept and components of Symfony UX',
```

Comment:
typo: => Concrete ex~~e~~**a**mple to understan~~t~~**d** all the concept**s** and components of Symfony UX